google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1alpha2/OnSchemaUpdateRunnable.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.bigquery.storage.v1alpha2;
+
+/**
+ * A abstract class that implements the Runable interface and provides access to the current
+ * JsonStreamWriter, StreamWriter, and updatedSchema. This runnable will only be called when a
+ * updated schema has been passed back through the AppendRowsResponse. Users should only implement
+ * the run() function. The following example performs a simple schema update.
+ *
+ * <p>Performing a schema update requires 2 steps: making a new connection with the same WriteStream
```

Comment:
I was thinking it would be hard for users to come up with what methods to use for schema updates unless they know in detail what the code is, so just wanted to put a case of schema update on there so users can build their own schema update based on the simple one. Should I still remove it?

Reply:
Users will usually have an idea of how this should be done, they will be developers for large connectors. You only need to make it clear why you implement it this way for JsonWriter.

Reply:
Got it.